pkg/client/proxy_snapshot.go

```diff
@@ -5,12 +5,10 @@ import (
 
 	etypes "github.com/longhorn/longhorn-engine/pkg/types"
 	eutil "github.com/longhorn/longhorn-engine/pkg/util"
+	ediskutil "github.com/longhorn/longhorn-engine/pkg/util/disk"
```

Comment:
Move const `VolumeHeadName` to another file like `"github.com/longhorn/longhorn-engine/pkg/types/types"` then we can use alias `enginetypes`

Reply:
Sound good. Thank you.